docs/source/api/v3/deliveryservices.rst

```diff
@@ -31,37 +31,39 @@ Request Structure
+	+=================+==========+=========================================================================================================================================+
+	| cdn             | no       | Show only the :term:`Delivery Services` belonging to the :ref:`ds-cdn` identified by this integral, unique identifier                   |
+	+-----------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------+
+	| id              | no       | Show only the :term:`Delivery Service` that has this integral, unique identifier                                                        |
+	+-----------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------+
+	| logsEnabled     | no       | Show only the :term:`Delivery Services` that have :ref:`ds-logs-enabled` set or not based on this boolean                               |
+	+-----------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------+
+	| profile         | no       | Return only :term:`Delivery Services` using the :term:`Profile` that has this :ref:`profile-id`                                         |
+	+-----------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------+
+	| tenant          | no       | Show only the :term:`Delivery Services` belonging to the :term:`Tenant` identified by this integral, unique identifier                  |
+	+-----------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------+
+	| type            | no       | Return only :term:`Delivery Services` of the :term:`Delivery Service` :ref:`ds-types` identified by this integral, unique identifier    |
+	+-----------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------+
+	| accessibleTo    | no       | Return the :term:`Delivery Services` accessible from a :term:`Tenant` *or it's children* identified by this integral, unique identifier |
+	+-----------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------+
+    | serviceCategory | no       | Show only the :term:`Delivery Services` belonging to the :term:`Service Category` identified by this integral, unique identifier        |
+    +-----------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
This table is malformed because you used spaces instead of tabs for indentation.